src/MiniCover/Reports/HtmlReport.cs

```diff
@@ -153,5 +154,11 @@ private string GetBgColor(ConsoleColor color)
                     throw new ArgumentException($"Invalid color: {color}");
             }
         }
+
+        private string GetHtmlFileName(string fileName)
+        {
+            string safeName = Regex.Replace(fileName, @"^[./\\]", "");
```

Comment:
I think linux path separator is different than /, I have to check

Reply:
Path.DirectorySeparatorChar and Path.AltDirectorySeparatorChar, but the documentation says they are hardcoded to be '/' and '\' (or vice versa). Note that there are two actual backslashes; they need to be escaped inside [].

Reply:
Oh, I am on Linux, by the way. Everything works great :)

Reply:
Can you please change the regex to @"^[./\\]+"    ?

Reply:
Here is some tests comparing both regexes:
https://dotnetfiddle.net/QAv7g8

Reply:
Umm, I don't know how that + (was *, I think) got lost, sorry. You are correct, of course. I amended the commit.

Reply:
Haha, now it makes sense. I was wondering why you're replacing only the first dot. 😄 
Thanks for updating it.